core/src/kmx/kmx_processevent.h

```diff
@@ -107,6 +106,7 @@ class KMX_ProcessEvent {
   KMX_Environment *GetEnvironment();
   KMX_Environment const *GetEnvironment() const;
   INTKEYBOARDINFO const *GetKeyboard() const;
+  void SetCapsLock(KMX_DWORD &modifiers, KMX_BOOL capsLockOn, KMX_BOOL force = FALSE);
```

Comment:
Why did you move this down? It seems to belong better where it was previously on line 89.

Reply:
I exposed the method it is now public. I could make a new public member that just calls the private SetCapsLock Method.

Reply:
I see

Reply:
No, it's fine to keep SetCapsLock public if that's what it needs to be.